doc/source/whatsnew/v0.24.0.txt

```diff
@@ -159,6 +161,24 @@ This is the same behavior as ``Series.values`` for categorical data. See
 :ref:`whatsnew_0240.api_breaking.interval_values` for more.
 
 
+.. _whatsnew_0240.enhancements.bubble_plots:
+
+Bubble plots
```

Comment:
Maybe "Automatic Marker Sizes" or something like that. Don't want to confuse people with "bubble" vs "scatter" plot.

Reply:
My suggestion : 
- For the title of the section in the whatnew entry we keep "bubble plots"
- the first sentence of that section starts with "Bubble plots (i.e. scatter plots in which individual marker sizes vary and reflect data) can now be produced..."

Reply:
In fact "bubble plots" already appear clearly in the main documentation in the visualization section. It probably does not matter too much how call these graphs in the whatsnew entry after all... I do as you suggest.